iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -470,55 +485,55 @@ Inspector.prototype.update = function () {
         }
         $('.duplicates .text').text((n + 1) + " of " + ndup);
         var viewer = this._viewer;
-        $('.duplicates .prev').off().click(function () { inspector.selectFact(duplicates[(n+ndup-1) % ndup].id)});
-        $('.duplicates .next').off().click(function () { inspector.selectFact(duplicates[(n+1) % ndup].id)});
+        $('.duplicates .prev').off().click(function () { inspector.selectItem(duplicates[(n+ndup-1) % ndup].id)});
+        $('.duplicates .next').off().click(function () { inspector.selectItem(duplicates[(n+1) % ndup].id)});
 
         this.getPeriodIncrease(cf);
 
     }
     else if (cf instanceof Footnote) {
         $('#inspector').addClass('footnote-mode');
+        $('#inspector .footnote-details .footnote-facts').html(this._footnoteFactsHTML());
     }
     this.updateURLFragment();
 }
 
 /*
- * Select a fact from the report.
+ * Select a fact or footnote from the report.
  *
- * Takes a fact ID to select and an optional list of "alternate" facts, which
- * will be presented in an accordian.  This is used when the user clicks on a
- * nested fact in the viewer, so that all facts corresponding to the area
- * clicked are shown.
+ * Takes an ID to select.  If the item is a fact, an optional list of
+ * "alternate" facts may be specified, which will be presented in an accordian.
+ * This is used when the user clicks on a nested fact in the viewer, so that
+ * all facts corresponding to the area clicked are shown.
  *
  * If factIdList is omitted, the currently selected fact list is reset to just
  * the primary fact.
  */
-Inspector.prototype.selectFact = function (id, factIdList) {
+Inspector.prototype.selectItem = function (id, factIdList) {
     if (factIdList === undefined) {
-        this._currentFactList = [ this._report.getFactById(id) ];
+        this._currentFactList = [ this._report.getItemById(id) ];
     }
     else {
         this._currentFactList = [];
         for (var i = 0; i < factIdList.length; i++) {
-            this._currentFactList.push(this._report.getFactById(factIdList[i]));
+            this._currentFactList.push(this._report.getItemById(factIdList[i]));
         }
     }
-    this.switchFact(id);
+    this.switchItem(id);
 }
 
 /*
- * Switches the currently selected fact.  Unlike selectFact, this does not
+ * Switches the currently selected fact.  Unlike selectItem, this does not
```

Comment:
should this terminology be updated to not be fact specific?

Reply:
Have attempted to clarify in e771d30, although your comment has made me realise that we probably ought to support nested footnotes, as well as facts nested inside footnotes and vice versa.

I'll tackle that separately.